implants/lib/eldritch/src/sys/get_ip_impl.rs

```diff
@@ -24,38 +27,48 @@ fn handle_get_ip() -> Result<Vec<NetInterface>> {
         for ip in network_interface.addr {
             ips.push(ip.ip());
         }
-        
-        res.push(NetInterface{
+
+        res.push(NetInterface {
             name: network_interface.name,
-            ips: ips,
+            ips,
             mac: mac_addr,
```

Comment:
Bet, let's refactor this section to use the short hand so update the name, and mac local variables to be the same as the dict params.